bicycleparameters/main.py

```diff
@@ -935,98 +1049,187 @@ def plot_eigenvalues_vs_speed(self, speeds, fig=None, generic=False,
+        #                     name='Imaginary',
+        #                     line=dict(color='royalblue',dash='dash'),
+        #                     text = 'Capsize'), row=1, col=1)
+         # fig.add_trace(go.Scatter(x=speeds, y=np.imag(evals),
+        #                     mode='lines',
+        #                     name='Imaginary',
+        #                     line=dict(color='royalblue',dash='dash'),
+        #                     text = 'Capsize'), row=1, col=2)
+        """Single plot""" 
+        """Somehow the real weave speeds give errors in the plot"""
+        
+        fig.add_trace(go.Scatter(x=speeds, y=np.real(wea1['evals']),
+                            mode='lines',
+                            name='Re',
+                            text = 'Weave'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.real(wea2['evals']),
+                            mode='lines',
+                            name='Re',
+                            text = 'Weave'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.real(cap['evals']),
+                            mode='lines',
+                            name='Re',
+                            text = 'Capsize'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.real(cas['evals']),
+                            mode='lines',
+                            name='Re',
+                            text = 'Castering'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.abs(np.imag(wea1['evals'])),
+                            mode='lines',
+                            name='Im',
+                            text = 'Weave'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.abs(np.imag(wea2['evals'])),
+                             mode='lines',
+                             name='Im',
+                             text = 'Weave'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.abs(np.imag(cap['evals'])),
+                            mode='lines',
+                            name='Im',
+                            # line=dict(color='royalblue',dash='dash'),
+                            text = 'Capsize'))
+        fig.add_trace(go.Scatter(x=speeds, y=np.abs(np.imag(cas['evals'])),
+                            mode='lines',
+                            name='Im',
+                            # line=dict(color='royalblue',dash='dash'),
+                            text = 'Capsize'))
+        vw = speeds[np.real(cas['evals'])<0][0]
+        vc = speeds[np.real(wea2['evals'])>0][0]
+        fig.add_vrect(x0=vw, x1=vc, 
+                      annotation_text="Self stability", annotation_position='top left',             
+                        fillcolor="green", opacity=0.25, line_width=0, row=1, col=1)
+        
+        fig.update_layout(title_text='Eigenvalues vs velocity',title_x=0.5,
+                   xaxis_title='Velocity [m/s]',
+                   yaxis_title='Eigenvalues')
+        fig.update_traces(hovertemplate= "%{x:.3f}<br>%{y:.3f}")
+
+        
+        # fig.show()
```

Comment:
Same thing for this one. Make a new method for the plotly plot, leaving the old one intact.